knative-serving-app/templates/knative-service.yaml

```diff
@@ -15,16 +15,26 @@ spec:
       containerConcurrency: 1
       containers:
       - args:
-        {{- if .Values.extraArgs }}
-{{ toYaml .Values.extraArgs | indent 8 }}
+          {{- if .Values.extraArgs }}
+          {{ toYaml .Values.extraArgs | indent 10 }}
```

Comment:
I suggest you to add an indent to anything inside `if/with/range` statement for more readability

Reply:
Done